pkg/operator/configobserver/network/observe_network.go

```diff
@@ -127,3 +127,26 @@ func validateCIDRs(in []string) error {
 	}
 	return nil
 }
+
+// GetServiceNodePortRange retrieves the ServiceNodePortRange for the cluster.
+func GetServiceNodePortRange(lister configlistersv1.NetworkLister, recorder events.Recorder) (string, error) {
```

Comment:
why public?

Reply:
this will get called from the cluster-kube-apiserver-operator's `ConfigObserver`.